src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/ShowCommand.java

```diff
@@ -0,0 +1,68 @@
+package seedu.address.logic.wanderlustlogic.wanderlustcommands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.wanderlustlogic.wanderlustcommands.exceptions.CommandException;
+import seedu.address.model.travelplanner.Model;
+
+public class ShowCommand extends Command {
+    public static final String COMMAND_WORD = "show";
+    public static final String ACTIVITY = "activity";
+    public static final String ACCOMMODATION = "accommodation";
+    public static final String FRIEND = "friend";
```

Comment:
Don't have to change now, but maybe we should store these constants under the respective classes (i.e. Activity, Accommodation and Friend class) as TPO_WORD, since we are referring to these strings in multiple other classes. Similar to COMMAND_WORD.

Reply:
Resolved.